include/library/spdm_requester_lib.h

```diff
@@ -24,6 +24,128 @@
  **/
 libspdm_return_t libspdm_init_connection(void *spdm_context, bool get_version_only);
 
+/**
```

Comment:
Since this is a new way to interact with libspdm messages these functions should go at, like, the bottom of this file. An Integrator will either use the classic way or this RESTful way and they shouldn't be mixed.

Reply:
Thanks. I have put the declarations to the bottom.